src/DeviceInterface/initFormSubmissionsApi.js

```diff
@@ -44,13 +45,13 @@ export function initFormSubmissionsApi (forms) {
         matchingForm?.submitHandler('global pointerdown event + matching form')
 
         if (!matchingForm) {
-            const selector = SUBMIT_BUTTON_SELECTOR + ', a[href="#"], a[href^=javascript], *[onclick]'
+            const selector = matching.cssSelector('submitButtonSelector') + ', a[href="#"], a[href^=javascript], *[onclick], [class*=button i]'
             // check if the click happened on a button
             const button = /** @type HTMLElement */(event.target)?.closest(selector)
             if (!button) return
 
-            const text = getText(button)
-            const hasRelevantText = /(log|sign).?(in|up)|continue|next|submit/i.test(text)
+            const text = getTextShallow(button) || extractElementStrings(button).join(' ')
```

Comment:
Renamed `getText` to `getTextShallow` because it tripped me. I thought it would loop through all descendants, but it stays one-level deep. Because of that, here I'm also ensuring adding `extractElementStrings` because of that.